lib/ostruct.rb

```diff
@@ -239,7 +239,11 @@ def freeze
         raise! ArgumentError, "wrong number of arguments (given #{len}, expected 1)", caller(1)
       end
       set_ostruct_member_value!(mname, args[0])
-    elsif len == 0
+    elsif len == 0 # and /\A[a-z_]\w*\z/ =~ mid #
+      if @table.key?(mid)
+        new_ostruct_member!(mid) unless frozen?
```

Comment:
Does did ever have any effect?

Reply:
Yeah, this likely does nothing since these aren't lazy loaded anymore! Thanks updating my PR.